core/ui/compose/common/src/main/kotlin/app/k9mail/core/ui/compose/common/mvi/UnidirectionalViewModel.kt

```diff
@@ -92,7 +91,7 @@ inline fun <reified STATE, EVENT, EFFECT> UnidirectionalViewModel<STATE, EVENT,
     val dispatch: (EVENT) -> Unit = { event(it) }
 
     LaunchedEffect(key1 = effect) {
-        effect.collectLatest {
+        effect.collect {
```

Comment:
Note: This seems to be …
a) … the right thing to do. We don't want to cancel handling the previous Effect when we receive another one.
b) … necessary so the permission screen is properly skipped on Android versions that don't support runtime permissions.

Reply:
I think the skip of the permission screen could be handled inside the navigation.